src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -95,18 +112,21 @@ export class AccessibleBufferWidget extends DisposableStore {
 			}
 		}));
 		this.add(this._xterm.raw.onWriteParsed(async () => {
-			if (this._accessibleBuffer.classList.contains(Constants.Active)) {
+			if (this._focusedContextKey.get()) {
 				await this._updateEditor(true);
 			}
 		}));
-		this._updateEditor();
 		this.add(this._bufferEditor.onDidFocusEditorText(async () => {
+			if (this._inQuickPick) {
+				return;
+			}
```

Comment:
Why is this needed? I wouldn't expect the editor to get focused when the quick pick is up, and if it was, wouldn't it hide?